roles/update-access/test/user2.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDWxIwWsWIxrG/yCSUXTp9zzumP9+IbUnefOjP+ggmkDbHOfIzGJCjI93ksNS+V3ekdgAWtUAMT+zoAcYmjAiFj4HanIispo1TCP7LsMIIGku3maoJduvAnBRWTMqaOFspmA1ndek7fzIRTM+tCr7G56TU/S00RjCvqndtrCnxUTqJ2A2/9h5dgI/5JrN8p+H+0qLCLObsdooYI8u5LtTykPWersk/MBnEDkKTeCFHYO7pAOfY65fZCuOLHKn7mccvR80nNFr9RTdtrxAIdo+9fd+V4HQF2qpynxXnXPJHo/RH1Mx/+2Lk5nDJ7DooDtZ7xXuUoS3K848RQaxJ+4kOR user2@test.lab
```

Comment:
Can we please name this file id_rsa_user2.pub (or something that indicates that it's the SSH public key)